README.rst

```diff
@@ -86,12 +86,13 @@ The default output is plain text of maximum width 79 characters.
 This can be adjusted using the ``--width`` parameter. To disable
 shortening altogether use ``--width=0``. The default width value
 can be saved in the config file as well. Use ``--format=wiki`` to
-enable simple MoinMoin wiki syntax. For stats which support them,
+enable simple MoinMoin wiki syntax or ``--format=wiki`` to enable
```

Comment:
```suggestion
enable simple MoinMoin wiki syntax or ``--format=md`` to enable
```

Reply:
Or, perhaps, would you mind if we use `--format=markdown` instead of the abbreviation?

Reply:
ok, I'll send a patch